antispam_bee.php

```diff
@@ -255,6 +255,20 @@ public static function init() {
 							'set_orderby_query',
 						)
 					);
+					add_action(
```

Comment:
Was meint Ihr, dieser gesamte Block lässt sich doch vereinfachen?
https://github.com/pluginkollektiv/antispam-bee/blob/302-filter-by-spam-reason/antispam_bee.php#L220-L271

*Beispiel:*
```
$edit_comments_hooks = [
	'filter' => [
		[
			'manage_edit-comments_columns' => 'register_plugin_columns',
			'manage_comments_custom_column' => 'print_plugin_column',
			'admin_print_styles-edit-comments.php' => 'print_column_styles',
			'manage_edit-comments_sortable_columns' => 'register_sortable_columns',
		]
	],
	'action' => [
			'pre_get_posts' => 'set_orderby_query',
			'restrict_manage_comments' => 'filter_columns',
			'pre_get_comments' => 'filter_by_spam_reason',
	]
];

foreach($edit_comments_hooks as $type => $edit_comments_hook ){
	foreach($edit_comments_hook as $hook => $val ){
		$args = ['Antispam_Bee_Columns', $val];
	
		if($type === 'filter'){
			add_filter($hook, $args);
		}else if($type === 'filter'){
			add_action($hook, $args);
		}
	}
};
```

Reply:
I really dislike the idea of adding actions and fitlers using an array like this. If we want to make the plugin code more readable, we should simply remove the many unnecessary line breaks that are used in most plugins. Then every add_filter/add_action is just a single line and not 7+ lines.

Reply:
@2ndkauboy y'r right but what about the redundant `Antispam_Bee_Columns` string

Reply:
It's just because all callbacks are static functions. Would you also consider it "redundant" if it would be a `$this` instead? I don't see that as an issue. Having just one giantic file with only static functions is the bigger issue for me ;)

Reply:
i think `$this` will not work. In this case `$this` is not the `Antispam_Bee_Columns` class.
But y'r right the hole file should refactored.

Reply:
Yes, `$this` would not work. I was just saying that I don't see the class name as redundant.

We should probably refactor the code base for version 3.0 to make it more readable.

Reply:
so then i'm quiet

Reply:
Regarding Refactoring: https://github.com/pluginkollektiv/antispam-bee/tree/version-3-0 I need to make this a PR for more visibility.